gh-pages-stub/public/css/styles.css

```diff
@@ -467,24 +467,81 @@ img {
     vertical-align: middle;
     height: auto;
 	max-width: 100%;
+	width: auto;
     margin: 20px auto;
     display: block;
+	max-height: 600px; /*for really big images*/
+}
+
+.featherlight img{
+	max-height: initial;
 }
 
 .content img{
 	cursor: pointer;
 }
 
+/* quote */
 blockquote{
+	padding: 15px 20px 15px 50px;
+	border-left: 5px solid #72cb66;
+    background: #eaf7e8;
+	position: relative;
+	font-size: 16px;
+}
+blockquote:before{
+    content: '';
+    position: absolute;
+    left: 5px;
+    top: 10px;
+    width: 40px;
+    height: 36px;
+    display: block;
+	background-image: url(/public/notes/note.png);
+}
+blockquote table{
+	outline: 2px solid #eaf7e8;
+}
+blockquote.tip{
+	border-left: 5px solid #4650ad;
+    background: #ecedf7;
+}
+blockquote.tip:before{
+	background-image: url(/public/notes/tip.png);
+}
+blockquote.tip table{
+	outline: 2px solid #ecedf7;
+}
+blockquote.important{
 	border-left: 5px solid #F7AD41;
     background: #FFF2DF;
 }
+blockquote.important:before{
+	background-image: url(/public/notes/important.png);
+}
+blockquote.important table{
+	outline: 2px solid #FFF2DF;
+}
+blockquote.caution{
+	border-left: 5px solid #f22f2f;
+    background: #feeaea;
+}
+blockquote.caution:before{
+	background-image: url(/public/notes/caution.png);
```

Comment:
@dr3amf4ll , https://github.com/kaaproject/kaa/pull/844 was just merged.
Please, update patch with next changes 
>  blockquote.important:before{
>  	background-image: url({{ site.baseurl }}/public/notes/important.png);
>  }
>  blockquote.important table{
>  	outline: 2px solid #FFF2DF;
>  }
>  blockquote.caution{
>  	border-left: 5px solid #f22f2f;
>      background: #feeaea;
>  }
>  blockquote.caution:before{
>  	background-image: url({{ site.baseurl }}/public/notes/caution.png);

And, please, check why this patch affects menu text align https://drive.google.com/file/d/0B6sIF8dhD0gVRXFMd3VaZTZqdTQ/view?usp=sharing